plugnparse/plugnparse/plugins.py

```diff
@@ -17,8 +17,7 @@ def scan_error_handler(self, name):
         self._scan_fails[name] = exc
         if not issubclass(exc[0], ImportError):
             raise  # reraise the last exception
-        # self._scan_fails.pop()
-
+        print(f'WARNING: Failed to scan: {name} due to import error')
```

Comment:
@brianthelion printing her rather than logging since its a CLI tool that prints already, however, let me know if you want me to change that to logging